lib/pry/config/behavior.rb

```diff
@@ -14,13 +14,17 @@ module Behavior
 
       module Builder
         #
-        # Returns a new Behavior, non-recursively (unlike {#from_hash}).
+        # @example
+        #   c = Pry::Config.assign(foo: {bar: {baz: 42}})
+        #   c.class # => Pry::Config
+        #   c.foo.class # => Hash
         #
         # @param
         #   (see #from_hash)
         #
-        # @return
-        #   (see #from_hash)
+        # @return [Pry::Config::Behavior]
+        #   Returns an initialized object who included {Pry::Config::Behavior},
+        #   without walking _attributes_ with recursion.
```

Comment:
I'm not sure if YARD makes underscores look like code but backpacks definitely work. You may want to change that but if `_` also means code, probably no need to bother.

Reply:
it's markdown (https://github.com/pry/pry/blob/master/.yardopts#L1) so i guess this appears _itatic_ rather than as `code`.

Reply:
I usually see parameters formatted as `code` but no biggie.

Reply:
switched to `code`  in the last update.